### ISSUE TITLE
Request: eye realignment for character swaps

### ISSUE THREAD

**Issue by NepposTheNose** [ScanMountGoat/xenoblade_blender#52]
**_Feature request: When exporting a model, I'd like the option to re-generate whatever file or parameter causes the issue described below._**

When swapping the face models for party members, their **_eyes (irises) become misaligned_** during the following events:
- using a camp
- using a canteen
- during cinematic cutscenes
- during Ouroboros Chain Orders after delinking

From that point onward, **_the eyes remain misaligned_** until the character's Costume is changed or the game is closed.
Because each party member's Costume is set to their default outfit when A-ing a campsite or canteen, their eyes automatically realign after leaving, but **_only_** when wearing another Costume.

This issue is **_not_** caused by xenoblade_blender, it happens when switching to any model.
Also happens when moving the eyes to the hair or Costume wimdo.
If I could "fix" the eyes in place I would, but I dunno how to.

![BUG CROSSEYED](https://github.com/user-attachments/assets/63af42bb-e307-4520-94b7-ea8ceda820e8)

**Comment by ScanMountGoat** [comment]
> This issue is not caused by xenoblade_blender

Does the alignment issue happen if you simply replace the files without modifying them in any way? Errors like this are helpful for figuring out the in game data, but I can't guarantee that this is fixable by simply patching this addon.

**Comment by NepposTheNose** [comment]
Yep, the issue is **not** caused by your plugin. It just happens when you swap one model with another, even unedited.

**Comment by NepposTheNose** [comment]
So it turns out... this was a simple case of skill issue, on my part!
**The fix is very simple: Realign the eye-bones in Blender's edit mode.**

I'm an absolute modeling noob, so it didn't even occur to me to try editing the skeleton until now.

For future reference, the **four bones** I realigned to fix Shania's eyeballs were: **FC_eye_R, FC_hi_R, FC_eye_L, FC_hi_L**
They point diagonally outward, so I just made them point straight forward and now Shania doesn't look dopey in camp anymore.

This is kind of a premature celebration – I haven't checked if it works with Guernica or Ouroboros Orders yet.
But I'm about to check.

**Comment by ScanMountGoat** [comment]
> Even swapping one player char with another (like changing all of Mio's model files with Glimmer's) causes this issue, though in that case, it was as if the irises "moved back into the skull". They moved depending on camera angle, which was fascinating.

I'll try and look into this later. I haven't looked into how the game actually animates character eye movements yet.
